### DISCUSSION TITLE
Laser 3018 Pro: Crashing during task (laser operation only)

### DISCUSSION THREAD

**Discussion by bradleyknigh** [arkypita/LaserGRBL#1221] [Q&A]
**Describe the bug**
Hello There,

I have recently brought 3018 Pro CNC Laser engraver. (beginning to wish I had never now)
Its always crashing halfway into a job and they are only small projects, some being 15-20 mins of Gcode
Its very random where it stops and I can not close laserGRBL software when it happens, the only way to stop the software is to unplug the data cable from the PC once this happens it closes the software down straight away.

Things I have already done to try to fix this issue :
 - I have replaced the standard controller board that came with 3018 PRO.
 - I have looked at using a different software. (does the same issue)
 - I got a shorter data cable to stop inference. 
 - Cleaned all components, disconnect every cable  and reconnected it back up.

Any ideas to what might be giving me my problems?
I'm all out of ideas.

Link to laser I brought: https://vevor.co.uk/products/cnc-3018-pro-15w-laser-engraver-cutter-engraving-machine-w-offline-controller?_pos=5&_sid=38c0b5844&_ss=r

**Hardware and software configuration (please complete the following information):**
 - Windows 10
 - Engraver 3018 Pro
 - Grbl version (1.1)
 - LaserGRBL version used both v3.0 and now v3.8.6

**Session Log When crash Happens**

![image](https://user-images.githubusercontent.com/76561606/103094142-7f006080-45f4-11eb-97a9-e2e0cee5e81f.png)

**Comment by arkypita** [comment]
I move this thread into "[discussions](https://github.com/arkypita/LaserGRBL/discussions)" because it is not a LaserGRBL issue, but a problem of your hardware.

**Comment by arkypita** [comment]
First of all: do you read [LaserGRBL faq](http://lasergrbl.com/faq/) about this issue?

By the side of LaserGRBL the only things you can do is trying to set a slower threading mode (LaserGRBL send messages to the engraver a little slower). Another problem could be that you have a too high step rate that the board is not able to handle (faq, point 3). Please copy your grbl configuration here. 

For me it is quite clear that the problem is related to the loss of communication by/from the machine. Maybe the USB plug is making a false contact, or the power supply is under-dimensioned.
